Fixed developer.mozilla.org documentation pages

Mertyn opened the pull request:
I fixed many icons on the docs, that didnt invert, but there may have been some I haven's seen and fixed yet.
Also I couln't fix the seach icon in the top right, maybe someone else knows how to best fix that.

![mdn](https://user-images.githubusercontent.com/22528516/103218291-e5e68800-491a-11eb-827a-c8b34fb62c42.jpg)

Gusted commented on src/config/dynamic-theme-fixes.config:
```diff
@@ -1917,9 +1917,17 @@ developer.mozilla.org
 INVERT
 mark
 .icon-thumbs-down-alt
-span[class*="bc-head-icon"]::before
```
This was specially crafted so all icons of any browser was inverted.

Mertyn commented:
I have added the request and while i was at it found some more icons that needed fixing. Additionally the logo had a problem, where it was already inverted on the home page, but not on the docs pages (before my fix), so i fixed that aswell.
